core/src/main/java/org/secondbase/core/SecondBase.java

```diff
@@ -43,6 +54,27 @@ private void init(final String[] args, final Flags flags) {
             flags.printVersion(System.out);
             System.exit(0);
         }
+
+        if (webConsole != null) {
+            try {
+                webConsole.start();
+            } catch (final IOException e) {
+                throw new SecondBaseException("Could not start webconsole.", e);
```

Comment:
Maybe we can use a more specific exception here? Isn't it that this really is a couldnotstartwebconsole exception?